relayer/relays/execution/main.go

```diff
@@ -154,14 +155,20 @@ func (r *Relay) Start(ctx context.Context, eg *errgroup.Group) error {
 				}
 
 				// ParentBeaconRoot in https://eips.ethereum.org/EIPS/eip-4788 from Deneb onward
-				err = beaconHeader.SyncExecutionHeader(ctx, *blockHeader.ParentBeaconRoot)
+				executionProof, err := beaconHeader.FetchExecutionProof(*blockHeader.ParentBeaconRoot)
 				if err == header.ErrBeaconHeaderNotFinalized {
 					logger.Warn("beacon header not finalized, just skipped")
 					continue
 				}
 				if err != nil {
-					return fmt.Errorf("sync beacon header: %w", err)
+					return fmt.Errorf("fetch execution header proof: %w", err)
 				}
+				inboundMsg.Proof.ExecutionProof = executionProof
+
+				logger.WithFields(logrus.Fields{
+					"EventLog": inboundMsg.EventLog,
+					"Proof":    inboundMsg.Proof,
+				}).Debug("Generated message from Ethereum log")
```

Comment:
Can we maybe log out the containing execution header (or beacon header) hash here? For easier debugging.

Reply:
Both are sub-fields of `Proof` so should already be in the log.